app/views/reservations/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <h2>Thank you for your order, <%= @reservation.user.username %>!</h2>
+  <p>Your order number is <strong><%= @reservation.id  %></strong></p>
+  <br>
+  <h4>Order Summary:</h4>
+  <div class="card mb-3">
+    <img class="card-img-top" src="..." alt="Meal-image">
+    <div class="card-body">
+      <h5 class="card-title"><%= @cooking_session.meal.name %></h5>
+      <p class="card-text">Prepared by <%= @cooking_session.meal.chef.user.username %></p>
```

Comment:
Can we use @cooking_session.chef_name
The chef_name method is defined in the CookingSession model